### PR TITLE
Add Profile to multi-tenancy example

### PR DESCRIPTION
**Pull request by gianlucam76**

### CODE REVIEW

**Comment by gianlucam76** [comment]
@egrosdou01 this very likely needs some work, but thought it was worth mentioning as it is an easy way to achieve multi-tenancy (with full isolation among tenant).

Let me know your feedbacks

File: docs/use_cases/use_case_multitenancy.md [hunk 1]
```diff
@@ -32,7 +32,15 @@ We mentioned two terms in the challenges above: `platform administrator` and `te
...
 - **Tenant administrator:** Is the persona that has admin access to the cluster or the namespaces where applications are running. The permissions are assigned by the platform administrator.
 
-## Sveltos Multitenancy Solution
+## Sveltos Multitenancy Solution: full isolation
+
+A common multi-tenant scenario involves assigning dedicated namespaces within the management cluster for each tenant. Tenant admins then create and manage clusters within their designated namespace and use Profile instances to define list of add-ons and applications to deploy in their managed clusters.
+
+Similar to ClusterProfiles, Profiles utilize a cluster selector and list of add-ons and applications. However,  Profiles operate within a specific namespace, matching only clusters created in that namespace. 
+
+![Profile vs ClusterProfile](../assets/profile_vs_clusterprofile.png)
```
Comment (egrosdou01) [inline docs/use_cases/use_case_multitenancy.md:41]: I would like to work on the diagram provided? What do you think?

Comment (gianlucam76) [reply docs/use_cases/use_case_multitenancy.md:41]: Thanks. Yes please go ahead. Thank you

Comment (egrosdou01) [reply docs/use_cases/use_case_multitenancy.md:41]: Sveltos_Profile_ClusterProfile

Comment (gianlucam76) [reply docs/use_cases/use_case_multitenancy.md:41]: I like it. Thank you.

Do you want to update this section adding your diagram? I can close this PR in that case

Comment (egrosdou01) [reply docs/use_cases/use_case_multitenancy.md:41]: I believe we need to keep the additions for the Sveltos Profile. We can merge the code changes and I will update the diagram later on.

**Pull request merged**